index.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"
+        crossorigin="anonymous">
+    <title>Newsweek Clone</title>
+</head>
+
+<body>
+
+    <header class="">
+        <nav class="">
+            <div class="container-xl top-menu">
+                <div class="row top-bar d-flex bg-danger align-items-center justify-content-lg-between">
+                    <div class="col-lg-4 d-none d-lg-block"><time class="text-white d-none d-lg-block"
+                            datetime="2021-01-20">Wed,
+                            Jan 20,
+                            2021</time></div>
+                    <div class="col-6 col-lg-4 text-center">
+                        <a href="#" title="icon"><img src="assets/icon.png" class="w-75" alt=""
+                                style="width: 150px;"></a>
+                    </div>
+                    <div class="col-6 col-lg-4 d-flex justify-content-end align-items-center">
+                        <a href="#" title="user" class="text-white ms-3"><i class="fa fa-user"
+                                aria-hidden="true"></i></a>
+                        <button type="button" class="btn btn-warning rounded fw-bold h6 ms-3 mt-2">SUBSCRIBE
+                        </button>
+                        <a href="#" title="menu" class="text-white ms-3"><i class="fa fa-bars"
+                                aria-hidden="true"></i></a>
+                    </div>
```

Comment:
- [ ] On the desktop screen, the user icon should be the text login and the hamburger icon should not be present. The hamburger and user icons appear on the medium and small screens.
Your clone desktop screen👇🏾 
<img width="1336" alt="Screen Shot 2021-03-01 at 8 10 22 PM" src="https://user-images.githubusercontent.com/48804258/109546191-26ed4b00-7aca-11eb-806c-1616a610cf4c.png">

Your clone mobile screen 👇🏾 
<img width="456" alt="Screen Shot 2021-03-01 at 8 13 01 PM" src="https://user-images.githubusercontent.com/48804258/109546468-851a2e00-7aca-11eb-89f3-f4e2b10c0162.png">


The original page desktop screen 👇🏾 
<img width="1341" alt="Screen Shot 2021-03-01 at 8 12 22 PM" src="https://user-images.githubusercontent.com/48804258/109546396-6f0c6d80-7aca-11eb-8501-34f7f486f439.png">

The original page mobile screen 👇🏾 
<img width="437" alt="Screen Shot 2021-03-01 at 8 13 37 PM" src="https://user-images.githubusercontent.com/48804258/109546514-9a8f5800-7aca-11eb-9bfd-aaf187f6a91c.png">